src/authenticated.js

```diff
@@ -1,17 +1,19 @@
 (function() {
     const onlyLoggedOut = document.querySelector('.only-loggedout'),
         onlyLogged = document.querySelector('.only-logged'),
-        onlyLoggedProfile = document.querySelector('.only-logged-profile');
+        onlyLoggedProfile = document.querySelector('.only-logged-profile'),
+        authReq = new XMLHttpRequest();
 
     // See if user is authenticated
-    fetch('/authenticated')
-    .then((resp) => resp.json())
-    .then((data) => {
+    authReq.open('GET', '/authenticated', true);
```

Comment:
Yup this code is working fine. 
But i am not sure about using fetch. its not supported in IE and Safari yet and not in mobile browser. [source](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Browser_compatibility)